README.md

```diff
@@ -27,7 +27,56 @@ These are the primary software versions used in production, and you should be ab
   - `brew doctor`
 
 
-### **2. Python**  
+### **2. (Mac) Install or update other dependencies:**
+
+```bash
+brew install libevent libmagic libxml2 libxslt openssl graphviz nginx
+brew install freetype libjpeg libtiff littlecms webp chromedriver
+brew tap petere/postgresql
+brew install postgresql@9.3
+brew link --force postgresql@9.3
+brew install --force node@6
+brew link node@6 --force
+brew cask install java8
+brew tap garrow/homebrew-elasticsearch17
+brew install elasticsearch@1.7
```

Comment:
@caseylitton shoudl we just remove references to elasticsearch1.7, rather than make another edit and PR?

Reply:
Yes, we should remove old install instructions.  

If a third party wants to use our repo pre es5 the readme exists as 1.7 in those tags.